lib/global.d.ts

```diff
@@ -20,6 +20,9 @@ interface Dictionary<T> {
 }
 
 declare module '*/package.json' {
```

Comment:
If you add `"resolveJsonModule": true` then you can remove this altogether as typescript will automatically generate the types for JSON files

Reply:
I tried it before, but it has side effects. Adding `"resolveJsonModule": true` to `tsconfig.json` somehow changes the build output location, which then breaks `npm run build`:
```
$ npm run build
...
{ Error: Cannot find module '/Users/paulo/balena/balena-io/balena-cli/build/actions/api-key.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at importCapitanoCommands (/Users/paulo/balena/balena-io/balena-cli/automation/capitanodoc/index.ts:56:18)
    at renderMarkdown (/Users/paulo/balena/balena-io/balena-cli/automation/capitanodoc/index.ts:46:8)
    at process._tickCallback (internal/process/next_tick.js:68:7) code: 'MODULE_NOT_FOUND' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! balena-cli@11.28.13 build:doc: `mkdirp doc/ && ts-node --transpile-only -P automation/tsconfig.json automation/capitanodoc/index.ts > doc/cli.markdown`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the balena-cli@11.28.13 build:doc script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

`resolveJsonModule` | Build output folder
--- | ---
false | `build/actions/api-key.js`
true | `build/lib/actions/api-key.js`

Reply:
That'll be because `package.json` is now part of the source files, which means the `tsc` build will include it in the output and hence bump everything down a level (ie `build/{package.json,src/}`)

Reply:
Thanks! 👍  I was guessing the reason would be something like that, and it's good to have confirmation. I might make this change in some other PR. It would need some extra testing -- like how the standalone zip package behaves with it.